framework/web/ErrorHandler.php

```diff
@@ -391,8 +391,8 @@ public function argumentsToString($args)
                 $args[$key] = '<span class="keyword">' . ($value ? 'true' : 'false') . '</span>';
             } elseif (is_string($value)) {
                 $fullValue = $this->htmlEncode($value);
-                if (mb_strlen($value, 'utf8') > 32) {
-                    $displayValue = $this->htmlEncode(mb_substr($value, 0, 32, 'utf8')) . '...';
+                if (mb_strlen($value, Yii::$app->charset) > 32) {
+                    $displayValue = $this->htmlEncode(mb_substr($value, 0, 32, Yii::$app->charset)) . '...';
```

Comment:
Do we need to hardcode UTF-8 as well on [line 165](https://github.com/yiisoft/yii2/blob/master/framework/web/ErrorHandler.php#L165)? It was there since @cebe commit https://github.com/yiisoft/yii2/commit/d4ad533d48773f90f3b89c550860a65986a10eca#diff-7a2fbae94c890a0d999b38f8a82fe81eR141